fedora_messaging/tests/unit/test_cli.py

```diff
@@ -76,17 +77,219 @@ def test_bad_conf(self, mock_consume):
             "Error: Invalid value: Configuration error: Failed to parse"
             " {}: error at line 1, column 1".format(BAD_CONF)
         )
-        runner = CliRunner()
-        result = runner.invoke(cli.cli, ["--conf=" + BAD_CONF, "consume"])
+        result = self.runner.invoke(cli.cli, ["--conf=" + BAD_CONF, "consume"])
         self.assertEqual(2, result.exit_code)
         self.assertIn(expected_err, result.output)
 
     @mock.patch("fedora_messaging.cli.api.consume")
     def test_missing_conf(self, mock_consume):
         """Assert a missing configuration file is reported."""
-        runner = CliRunner()
-        result = runner.invoke(cli.cli, ["--conf=thispathdoesnotexist", "consume"])
+        result = self.runner.invoke(cli.cli, ["--conf=thispathdoesnotexist", "consume"])
         self.assertEqual(2, result.exit_code)
         self.assertIn(
             "Error: Invalid value: thispathdoesnotexist is not a file", result.output
         )
+        """
+        @click.option("--app-name", help=_app_name_help)
+        @click.option("--callback", help=_callback_help)
+        @click.option("--routing-key", help=_routing_key_help)
+        @click.option("--queue-name", help=_queue_name_help)
+        @click.option("--exchange", help=_exchange_help)
+        @click.option("--amqp-url", help=_amqp_url_help)
+        """
```

Comment:
I don't think this does anything. Did you mean to put this here?

Reply:
I forgot to remove my "helpfull lines" :)